app/services/search_api.rb

```diff
@@ -9,11 +9,11 @@ def initialize(organisation_id, scheme_id)
 
   def call
     case @scheme_id
-    when 'GB-COH'
```

Comment:
We should start moving any hard coded into configs or in separate constants file (SoC)